src/store/modules/recipeStore.js

```diff
@@ -107,5 +121,44 @@ export const recipeStore = {
         alert("There was an error creating the recipe");
       }
     },
+
+    async update_recipe(context, payload) {
```

Comment:
I don't think it's necessary to make these calls async, let's just avoid them altogether. In this use case more so, there is no point in making the call synchronously.

Reply:
This has been changed